src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -829,7 +832,8 @@ public String run(final Matcher m) {
 		Functions.clearFunctions(f); // Functions are still callable from other scripts
 		// We're just making it impossible to look them up
 		try {
-			String name = Skript.getInstance().getDataFolder().toPath().resolve(Skript.SCRIPTSFOLDER).relativize(f.toPath()).toString();
+			Path scriptsFolder = Skript.getInstance().getDataFolder().toPath().toAbsolutePath();
+			String name = scriptsFolder.resolve(Skript.SCRIPTSFOLDER).relativize(f.toPath().toAbsolutePath()).toString();

```

Comment:
If `scriptsFolder` isn't used elsewhere, why did you split it up? (I guess it doesn't matter since the complier will take it out but still)